test/parse_test.js

```diff
@@ -51,11 +50,6 @@ describe('parse()', function() {
     const result = await parser.parse(inputYAML, { path: __filename });
     await expect(JSON.stringify(result.json())).to.equal(outputJSON);
   });
-
-  it('should not apply traits', async function() {
-    const result = await parser.parse(inputYAML, { path: __filename, applyTraits: false });
-    await expect(JSON.stringify(result.json())).to.equal(outputJsonNoApplyTraits);
-  });
```

Comment:
Is there a reason this test is removed?

Reply:
It's not removed, its moved down with new test case.

Reply:
Ahh my bad, missed that yesterday 🧐